migrations/sql/afterMigrate__99__create_test_db.sql

```diff
@@ -1,2 +1,13 @@
-DROP DATABASE IF EXISTS mds_test;
-CREATE DATABASE mds_test WITH TEMPLATE mds OWNER mds;
\ No newline at end of file
+DO
+$do$
+DECLARE
+BEGIN
+  CREATE EXTENSION IF NOT EXISTS dblink;
```

Comment:
FYI https://www.postgresql.org/docs/9.5/postgres-fdw.html is the new way and 'create extension postgres_fdw' is available out of the box.

Reply:
it's what oracle_fdw is based upon..  which is part of our postgres build

Reply:
yes! that works great. 
We don't do anything with the fdw in our test schema, so that should stay unaffected for now. :D